Sources/_OpenAPIGeneratorCore/Extensions/SwiftFormat.swift

```diff
@@ -11,27 +11,27 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
+import Foundation
 import SwiftFormat
 import SwiftFormatConfiguration
 
 extension String {
+    private static let configurationURL = {
+        let configurationFilePath = #file
+            .components(separatedBy: "/")
+            .prefix(while: { $0 != "Sources" })
+            .joined(separator: "/")
+            .appending("/.swift-format")
```

Comment:
I feel like this is quite susceptible to be triggered by other folders that have a `Sources` in their name. I started with it because it seemed the simplest. What is your opinion, am I being overly cautions here?

Maybe something like this might work better as it narrows down the search criteria?
```suggestion
        let configurationFilePath = #file
            .components(separatedBy: "swift-openapi-generator/Sources")[0]
            .appending("swift-openapi-generator/.swift-format")
```

Or is there some other way of getting the file location, that I am missing?